Makefile

```diff
@@ -46,11 +46,15 @@ build-cli-linux: ## Build the Linux CLI
 build-cli-mac: ## Build the Mac CLI
 	cd cli && $(MAKE) build-mac
 
+build-bootstrap-registry:
+	cd cli && $(MAKE) build-bootstrap-registry
+
 build-cli: clean build-cli-linux build-cli-mac ## Build the CLI
 
-init-package: ## Create the zarf init package, macos "brew install coreutils" first
+init-package: build-bootstrap-registry ## Create the zarf init package, macos "brew install coreutils" first
 	$(ZARF_BIN) package create --confirm
 	mv zarf-init.tar.zst build
+	rm -f build/zarf-registry
```

Comment:
I have mixed feelings about this.

I felt like the fact that you could do a `zarf package create --confirm` at the main directory level to create the init package was a cool 'feature'. It also makes the init zarf.yaml conform with how we want our other example yamls to be (but as writing this I realize we do have some temporary Makefile abstraction for kustomize building resources for the BigBang example- but I think that's different because that's something that's more obvious since we're not removing the kustomzie files afterwards). 

The fact that the logic for handling the zarf-registry binary is in the `packager/create.go` and not referenced in the init zarf.yaml itself now means that someone who is creating package is forced to either use the Make command or to dive into source code to figure out why they're getting an error about some `unable to copy file` error for a file not referenced in the yaml. 

This will also make it harder for people to write their own custom init zarf.yaml in the future since 
1. it's something the CLI is expecting but it is not defined as part of the init schema.   #This is my biggest concern
2. It means that they'll have to clone the repo (or have their own registry.go to compile) instead of writing a yaml pointing to their own resources. (I'm not 100% sure this statement is true but I think it is..)


The creation now also depends on the busybox binary and the init.sh but since those assets aren't being explicitly removed I don't have as many concerns about that.

I'm not sure I worded all my thoughts on this well enough, does this make sense @jeff-mccoy?